lib/config/openssh/openssh.go

```diff
@@ -35,7 +35,18 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
-var sshConfigTemplate = template.Must(template.New("ssh-config").Parse(
+var sshConfigTemplate = template.Must(template.New("ssh-config").Funcs(template.FuncMap{
+	// shellQuote prepares a string for insertion into the ssh_config
+	"shellQuote": func(s string) string {
```

Comment:
Unless you want to split this into two functions, one for the shell quoting and the other for the percent escaping, i think we should make it clear that this isn't just shell quoting:

```suggestion
	// proxyCommandQuote prepares a string for insertion into the ssh_config
	"proxyCommandQuote": func(s string) string {
```

Reply:
00e78dffeb